transaction/errors.go

```diff
@@ -79,15 +77,14 @@ func ErrorLogEncoder(getLogger GetLoggerFunc, ee kithttp.ErrorEncoder) kithttp.E
 		}
 		logger := getLogger(ctx)
 		if logger != nil && code != http.StatusNotFound {
-			logger.Log("sending non-200 response, non-404 response", level.Key(), level.ErrorValue(),
-				logging.ErrorKey(), e.Error(), "tid", ctx.Value(ContextKeyRequestTID).(string),
+			logger.Info("sending non-200 response, non-404 response", zap.String("error", e.Error()),
+				zap.String("tid", ctx.Value(ContextKeyRequestTID).(string)),
 			)
 		}
 		ee(ctx, e, w)
 	}
 }
 
-func GetLogger(ctx context.Context) kitlog.Logger {
-	logger := kitlog.With(logging.GetLogger(ctx), "ts", kitlog.DefaultTimestampUTC)
-	return logger
+func GetLogger(ctx context.Context) *zap.Logger {
+	return ctx.Value(1).(*zap.Logger)
```

Comment:
I don't think this will work.  We need a key rather than just an integer. I'm not sure how this is being used (usually getlogger and setlogger functions are together and the keys match), but I believe we want to be getting the logger that is being set in the bascule auth chain.